client/src/components/conclusions-list.tsx

```diff
@@ -0,0 +1,175 @@
+          InputLabelProps={{
+            shrink: true,
+          }}
+          value={conclusion || ""}
+          onChange={(e) => {
+            handleConclusionChange(e.target.value);
+          }}
+          variant="outlined"
+        />
+        <CardActions>
+          {canDelete ? (
+            <IconButton
+              id="delete-conclusion"
+              aria-label="remove conclusion"
+              size="small"
+              onClick={handleRemoveConclusion}
+            >
+              <ClearOutlinedIcon />
+            </IconButton>
+          ) : null}
+        </CardActions>
+      </CardContent>
+    </Card>
+  );
+};
+
+const ConclusionsList = (props: {
+  classes: any;
+  conclusions: string[];
+  updateConclusions: (val: string[]) => void;
+}) => {
+  const { classes, conclusions, updateConclusions } = props;
+
+  function onDragEnd(result: DropResult) {
+    if (!result.destination) {
+      return;
+    }
+    const startIdx = result.source.index;
+    const endIdx = result.destination.index;
+    const [removed] = conclusions.splice(startIdx, 1);
+    conclusions.splice(endIdx, 0, removed);
+    updateConclusions([...conclusions]);
+  }
+
+  function handleConclusionChange(val: string, idx: number): void {
+    conclusions[idx] = val;
+    updateConclusions([...conclusions]);
+  }
+
+  function handleAddConclusion(): void {
+    conclusions.push(
+      "Add a conclusion statement, e.g. 'In summary,  RGB colors are red, green, and blue'"
+    );
+    updateConclusions([...conclusions]);
+  }
+
+  function handleRemoveConclusion(index: number): void {
+    conclusions.splice(index, 1);
+    updateConclusions([...conclusions]);
+  }
+
+  return (
+    <Paper elevation={0} style={{ textAlign: "left" }}>
+      <Typography variant="body2" style={{ padding: 15 }}>
+        Conclusions
+      </Typography>
+      <DragDropContext onDragEnd={onDragEnd}>
+        <Droppable droppableId="droppable">
+          {(provided, snapshot) => (
+            <List
+              {...provided.droppableProps}
+              ref={provided.innerRef}
+              className={
+                snapshot.isDraggingOver ? classes.listDragging : classes.list
+              }
+            >
+              {conclusions.map((row, i) => (
+                <Draggable
+                  key={`conclusion-${i}`}
+                  draggableId={`conclusion-${i}`}
+                  index={i}
+                >
+                  {(provided, snapshot) => (
+                    <ListItem
+                      ref={provided.innerRef}
+                      {...provided.draggableProps}
+                      {...provided.dragHandleProps}
+                    >
+                      <ConclusionCard
+                        idx={i}
+                        conclusion={row}
+                        canDelete={conclusions.length > 1}
+                        handleConclusionChange={(val: string) => {
```

Comment:
might be not necessary here but for performance reasons (to avoid rerenders) can be better to wrap functions passed to components as params in `useCallback` hook: https://reactjs.org/docs/hooks-reference.html#usecallback